assets/i18n/en_US.json

```diff
@@ -241,15 +240,15 @@
         "importPatchesHint": "Import patches selection from a JSON file",
         "importedPatches": "Patches selection imported",
 
-        "resetStoredPatchesLabel": "Reset patches",
+        "resetStoredPatchesLabel": "Reset patches selection",
```

Comment:
Im not sure if it should be "Reset patches selection" or "Reset patch selection" (same question for the similar options)

Reply:
Maybe "Reset selection of patches"?

Reply:
I think that sounds clunky, and you'd need to change to that wording format for the other options for consistency 

(Ie: Export selection of patches)

Reply:
What do you think @PalmDevs

Reply:
Technically, it should be `patch selections`, since it is multiple selection*s* of patches. This can be combined with options which would result in something along the lines of:
```
Reset patch selections and options
```

`Patches selection` just doesn't sound right in my opinion.

Reply:
Okay, ill make these changes today

Reply:
![image](https://github.com/ReVanced/revanced-manager/assets/13122796/44b3fa18-ae1f-4301-a44a-d76a0db01ca7)